apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/MapEntryAssertionSamples.kt

```diff
@@ -0,0 +1,47 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class MapEntryAssertionSamples {
+
+    @Test
+    fun isKeyValue() {
+        expect(mapOf(1 to "a").entries.first()).isKeyValue(1, "a")
+
+        fails {
+            expect(mapOf(1 to "a").entries.first()).isKeyValue(1, "b")
+        }
+    }
+
+    @Test
+    fun key() {
+        expect(mapOf(1 to "a").entries.first())
+            .key {
```

Comment:
Please have a look at ListAssertionSamples -> get (line ~45) please add similar comments here and below to outline what happens.